action.yml

```diff
@@ -5,14 +5,14 @@ inputs:
   main-branch-name:
     description: 'The name of the main branch in your repo, used as the target of PRs. E.g. main, master etc'
     default: 'main'
-  tag-match-pattern:
-    description: 'The glob(7) pattern to be provided to `git describe --match` in order to match against the latest relevant tag on the specified main branch'
-    default: 'nx_successful_ci_run*'
   set-environment-variables-for-job:
     description: 'Applies the derived SHAs for base and head as NX_BASE and NX_HEAD environment variables within the current Job'
     default: 'true'
-  error-on-no-matching-tags:
-    description: 'By default, if no matching tags are found on the main branch to determine the SHA, we will log a warning and use HEAD~1. Enable this option to error and exit instead.'
+  error-on-no-successful-workflow:
+    description: 'By default, if no successful workflow is found on the main branch to determine the SHA, we will log a warning and use HEAD~1. Enable this option to error and exit instead.'
+  workflow-id:
```

Comment:
I don’t think we should need this option at all, we can retrieve the current workflow name from the environment/github global object (which we currently use to get the event_name)

Reply:
We could leave it as an optional override I guess although I’m struggling to imagine what workflow would require this to reference something other than the current workflow as a reference point

Reply:
I haven't found a way how to retrieve the workflow id from the github context/environment. The only way I found was to fetch all project's `workflows` and then look for the workflow whose `name` matches `$GITHUB_WORKFLOW` (or `github.workflow` if you prefer github context). I would not recommend this.